ores/score_processors/score_processor.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, scoring_contexts, score_cache=None,
         self.metrics_collector = metrics_collector or Null()
 
     def score(self, context, model, rev_ids, caches=None,
-              include_features=False, precache=False):
+              include_features=False, precache=False, features_cache={}):
```

Comment:
Are you sure? I'm trying with _process right now and its cache variable was this: {<datasource.revision.user.info.registration>: Timestamp('2012-10-30T16:18:14Z'), <datasource.revision.text>: '{"type":"item","id":"Q634","labels":{"nb":{"language":"nb","value":"Planet"}},"descriptions":{"nb":{"language":"nb","value":"Himmellegeme"}},"aliases":[],"claims":[],"sitelinks":{"enwiki":{"site":"enwiki","title":"Planet","badges":[]},"nowiki":{"site":"nowiki","title":"Planet","badges":[]},"nnwiki":{"site":"nnwiki","title":"Planet","badges":[]}}}', <datasource.revision.timestamp>: Timestamp('2012-10-30T17:39:00Z'), <datasource.revision.comment>: '/* wbsetsitelink-set:1|nnwiki */ Planet', <datasource.revision.user.info.groups>: {'autoconfirmed', '*', 'user'}, <datasource.revision.parent.text>: '{"type":"item","id":"Q634","labels":{"nb":{"language":"nb","value":"Planet"}},"descriptions":{"nb":{"language":"nb","value":"Himmellegeme"}},"aliases":[],"claims":[],"sitelinks":{"enwiki":{"site":"enwiki","title":"Planet","badges":[]},"nowiki":{"site":"nowiki","title":"Planet","badges":[]}}}', <datasource.revision.user.id>: 4755}